src/docs/release-notes/R27/FullNotes.mdx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+## Component Libraries
+
+-   Fixed 6 issues in Angular component library.
+-   Fixed 8 issues in React component library.
+-   Fixed 9 issues in React Native component library.
```

Comment:
We actually only fixed 1 this quarter I think...the rest of the milestone was done last quarter...buuuuut I'm fine with leaving this as-is.